src/gdal/ogr_file_format.cpp

```diff
@@ -1176,15 +1176,9 @@ Object* OgrFileImport::importPointGeometry(OGRFeatureH feature, OGRGeometryH geo
 		return object;
 	}
 	
-	if (symbol->getType() == Symbol::Text)
+	if (symbol->getType() == Symbol::Text && !symbol->getAuxiliaryProperty(QLatin1String("label")).isNull())
 	{
-		const auto& description = symbol->getDescription();
-		auto length = description.length();
-		auto split = description.indexOf(QLatin1Char(' '));
-		FILEFORMAT_ASSERT(split > 0);
-		FILEFORMAT_ASSERT(split < length);
-		
-		auto label = description.right(length - split - 1);
+		auto label = symbol->consumeAuxiliaryProperty(QLatin1String("label")).toString();
```

Comment:
I didn't test now, but I guess that this shows the potential *and* the missing pieces:
Putting the label into the description was a hack, but it helped to assign objects with the same label to a a single symbol. Probably also on a second and third import into the same map. 

This could be covered with an automated test: a test vector file with to identical objects, imported twice. At the end there must be a map with four objects with one symbol.

Reply:
Due to my understanding of OgrFileImport::getSymbolForLabel() objects with the same color and font size are assigned to a single symbol, the label as well as anchor and angle are always overwritten.
My proposal would not change the current behaviour, IMO.

Reply:
Done